### DISCUSSION TITLE
Welcome to sd_dreambooth_extension Discussions!

### DISCUSSION THREAD

**Discussion by d8ahazard** [d8ahazard/sd_dreambooth_extension#71] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mykeehu** [comment]
I just want to thank you for the fast and efficient development, the complement that you made and this is my latest toy. Now everything works, I am happy! Thank you!
